source/ui/data/VerticalDelegate.js

```diff
@@ -191,7 +207,8 @@
 			metrics.height = this.pageHeight(list, page);
 			metrics.width  = this.pageWidth(list, page);
 			// update the childSize value now that we have measurements
-			this.childSize(list);
+			var oldChildSize = list.childSize;
+			list.isChildSizeUpdated = (oldChildSize == this.childSize(list)) ? false : true;
```

Comment:
Would it be better to move this, and related calculations, to `refresh`, so that we don't perform these calculations unnecessarily upon each page generation? From the app side, it seems that the developer is calling `refresh`, necessarily, when they know they have updated the size of the children.